app/components/osf-footer/template.hbs

```diff
@@ -0,0 +1,34 @@
+<footer class="Footer m-t-lg m-b-lg p-lg bg-color-light">
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-2 col-md-3 col-md-offset-1">
+                <h4>{{t 'general.OSF'}}</h4>
+                <ul>
+                    <li class='Footer__link-item'><a href="https://status.cos.io/">{{t 'footer.status'}}</a></li>
+                    <li class='Footer__link-item'><a href="{{supportEmail}}" rel="nofollow">{{t 'footer.contact'}}</a></li>
+                    <li class='Footer__link-item'>{{#link-to 'support'}}{{t 'footer.faq'}}{{/link-to}}</li>
+                    <li class='Footer__link-item'><a href="https://api.osf.io/v2/docs/">{{t 'general.api'}}</a></li>
+                    <li class='Footer__link-item'><a href="https://github.com/CenterForOpenScience/osf.io">{{t 'footer.source_code'}}</a></li>
```

Comment:
All these links fail the color contrast checking :(

Reply:
@hmoco Might be easiest to lighten up the gray a bunch. Maybe try a couple of things and send screenshots to the product consultation flow for approval?

Reply:
So, initially I didn't change that because I figured we would want the footer to not change colors as we moved from ember to non-ember pages. But trying it out now, I'm not convinced I know what's wrong here. Making the background anything but exactly white still creates color contrast errors. Could the checker be wrong? Or do normal links just don't contrast well with anything but white? Ideas?
<img width="906" alt="screen shot 2018-03-15 at 11 14 48 am" src="https://user-images.githubusercontent.com/6268982/37472442-7d66b1ca-2842-11e8-84ff-265ebbda2bcd.png">

Reply:
@brianjgeiger

Reply:
@hmoco I think also try a darker blue for the links

Reply:
We have also, in other places, made the links a heavier typeface.

Reply:
👍 for bold links and/or darker font color. Since the footer on all of the pages, including outside this repo, we should consider making visible changes in a separate PR. That way, all of the changes can be coordinated, but in the mean time we get the footer back in ember-osf-web.

We can update ember-osf then have PRs ready to go for the dependent repos as well as PRs ready for osf.io and ember-osf-web. Not saying that you should have to make those PRs, but that's what would need to happen to avoid the footer looking different when the user clicks around the site.